modules/cluster-logging-collector-log-forwarding-about.adoc

```diff
@@ -127,17 +128,18 @@ spec:
 ** Specify the URL and port of the Kafka broker as a valid absolute URL, including the prefix.
 <6> Configuration for an input to filter application logs from the `my-namespace` project.
 <7> Configuration for a pipeline to send audit logs to the secure external Elasticsearch instance:
-** Optional: For `name`, you can give a name that describes the pipeline.
-** For `inputRefs`, name one or more log types as the inputs for this pipeline.
-** For `outputRefs`, provide the name of one or more outputs to use. For example, `elasticsearch-secure` forwards the audit log data to a secure Elasticsearch instance and `default` forwards the data to the internal Elasticsearch instance.
-** Optional: For `labels` you can specify labels to add to the logs.
-<8> Configuration for a pipeline to send infrastructure logs to  the insecure external Elasticsearch instance:
-<9> Configuration for a pipeline to send logs from the `my-project` project to the internal Elasticsearch instance.
+** Optional. A name to describe the pipeline.
+** The `inputRefs` is the log type, in this example `audit`.
+** The `outputRefs` is the name of the output to use, in this example `elasticsearch-secure` to forward to the secure Elasticsearch instance and `default` to forward to the internal Elasticsearch instance.
+** Optional: Labels to add to the logs.
+<8> Optional: Forward structured JSON log entries as JSON objects in the `structured` field. The log entry must contain valid structured JSON; otherwise, OpenShift Logging removes the `structured` field and instead sends the log entry to the default index, `app-00000x`.
+<9> Configuration for a pipeline to send infrastructure logs to  the insecure external Elasticsearch instance:
```

Comment:
Should this sentence end with a period(.) rather than a colon(:) ?

Reply:
Updated. Thank you.